action/protocol/execution/evm/evm.go

```diff
@@ -236,9 +237,12 @@ func executeInEVM(evmParams *Params, stateDB *StateDBAdapter, gasLimit uint64) (
 		ret, remainingGas, evmErr = evm.Call(executor, *evmParams.contract, evmParams.data, remainingGas, evmParams.amount)
 	}
 	if evmErr != nil {
-		log.L().Debug("evm error", zap.Error(err))
-		if err == vm.ErrInsufficientBalance {
-			return nil, evmParams.gas, remainingGas, action.EmptyAddress, true, evmErr
+		log.L().Debug("evm error", zap.Error(evmErr))
+		// The only possible consensus-error would be if there wasn't
+		// sufficient balance to make the transfer happen.
+		// Should be a hard fork (Bering)
+		if evmErr == vm.ErrInsufficientBalance && isBering {
```

Comment:
This logic seems to be changed wrongly? Before the change, if `evmErr == vm.ErrInsufficientBalance`, we will return error. After the change, if before isBering height, there will return error at all?